modules/serverless-functions-specifying-function-revision.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-on-cluster-builds.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-functions-specifying-function-revision_{context}"]
+= Specifying function revision
+
+When building and deploying a function on the cluster as opposed to locally, you need to point to the correct function revision by specifying the Git repository, branch, location within the repository, and other parameters. In addition to specifying these parameters using the `func.yaml` configuration file, you can use the flags for the `kn func deploy` command.
+
+.Prerequisites
+
+* {pipelines-title} must be installed on your cluster.
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have installed the Knative (`kn`) CLI.
+
+.Procedure
+
+* Deploy your function:
++
+[source,terminal]
+----
+$ kn func deploy --remote \
+                 --build git \
+		 --git-url <repo-url> \
+                 --git-branch <branch> \
```

Comment:
Can these flags line up?

Reply:
@lance Line up meaning go into a single line? I'd rather not:
* Per Ashleigh's suggestion, the options now have numbered callouts, which work better with multi-line listing. (I don't think they work in-line at all, in fact)
* The example at the end contains a non-short URL, so IMO it's better to split & align the individual flags.

Feel free to push back again though & I'll change it to a single line. FWIW though, copypasting this example works nicely in ksh and bash and probably most other shells.

Reply:
I actually meant vertical alignment like this.

```
$ kn func deploy --remote \
                 --git-url <repo-url> \
                 --git-branch <branch>
```
Also, note here you don't need the `--build git` flag.

Reply:
Thanks! `--build git` removed and tabs -> spaces fixed. :man_facepalming: